libzecale/serialization/proto_utils.hpp

```diff
@@ -11,12 +11,13 @@
 namespace libzecale
 {
 
-template<typename ppT, typename snarkT, typename snarkHandlerT>
-transaction_to_aggregate<ppT, snarkT> transaction_to_aggregate_from_proto(
+template<typename ppT, typename apiHandlerT>
+transaction_to_aggregate<ppT, typename apiHandlerT::snarkT>
```

Comment:
Why do we have `apiHandler::snarkT` and `wVerifierT::SnarkT` (i.e. why do we have "small and big" `S` for snark)?

Reply:
addressed above.  all types use lower-case, consistent with classes etc.  `T` for type parameters only.